### ISSUE TITLE
DeepLabV3+ (tensorflow)utilizes decoder features

### ISSUE THREAD

**Issue by LingyuZhuFin** [jfzhang95/pytorch-deeplab-xception#92]
Notice that this pytorch version is using different decoder_output_stride=4 features from deeplabv3+ tensorflow https://github.com/tensorflow/models/tree/master/research/deeplab ('entry_flow/block2/unit_1/xception_module/separable_conv2_pointwise') which its filter number is 256 instead of 128 here.

Just for letting you guys know, think this might affect the final prediction as well.
Also please let me know if you finally figure out this is not true.

**Comment by PTL2011** [comment]
Hi, would you mind sharing your training parameters and tricks?  My training result is horrible with the backbone of xception, I can't figure out why. These are mine:backbone='xception', base_size=513, batch_size=12, checkname='deeplab-xception', crop_size=513, cuda=True, dataset='pascal', epochs=50, eval_interval=1, freeze_bn=False, ft=False, gpu_ids=[0, 1, 2, 3], loss_type='ce', lr=0.01, lr_scheduler='poly', momentum=0.9, nesterov=False, no_cuda=False, no_val=False, out_stride=16, resume=None, seed=1, start_epoch=0, sync_bn=True, test_batch_size=12, use_balanced_weights=False, use_sbd=True, weight_decay=0.0005, workers=4
